NEWS.rst

```diff
@@ -63,6 +63,7 @@ style has been reformatted with the ``black`` tool.
 Many thanks to the Biopython developers and community for making this release
 possible, especially the following contributors:
 
+- Andrey Raspopov
```

Comment:
I think this will be the first Biopython release you've contributed to, so you can add " (first contribution)" here too.